projects/storefrontapp-e2e-cypress/cypress/e2e/vendor/asm/b2b/customer-list.e2e.cy.ts

```diff
@@ -20,6 +20,20 @@ context('Assisted Service Module', () => {
     Cypress.env('BASE_SITE', ELECTRONICS_BASESITE);
   });
   describe('ASM Customer list', () => {
+    it('Should be able to create a b2b cart via API (CXSPA-1595)', function () {
```

Comment:
Since this part is more like a prerequisite other than a test. I prefer to extract this test to a function, maybe named as 'prepareCartWithProduct()'. And then you can call the function in the following test case 'checking custom list features'.

Reply:
Done